vulcanai2/datasets/tabulardataset.py

```diff
@@ -56,8 +57,11 @@ def __getitem__(self, idx):
         """
         # Where df.drop is used to access the dataframe without the label column, iloc gets the row, then access values
         # and convert
+
         xs = self.df.drop(self.labelColumn, axis=1).iloc[[2]].values.tolist()[0]
-        y = self.df[[self.labelColumn]].iloc[[idx]].values.tolist()[0]
+        xs = torch.tensor(xs, dtype=torch.float)
+        y = self.df[[self.labelColumn]].iloc[[idx]].values.tolist()[0][0]
+        y = torch.tensor(y, dtype=torch.long)
```

Comment:
would we be able to get the type of y and use that to inform if we should use `dtype=torch.float` or `dtype=torch.long` @joemehltretter ?